src/core-manager/compat.js

```diff
@@ -0,0 +1,17 @@
+import { createRequire } from 'module'
+
+const require = createRequire(import.meta.url)
+
+// Export the appropriate version of `quickbit-universal` as the plain import
+// may resolve to an older version in some environments
```

Comment:
purely out of curiosity, what's this about?

Reply:
Don't actually know. This is copied from hypercore code